kokoro/macos_external/continuous.cfg

```diff
@@ -0,0 +1,6 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+
+# Location of the bash script. Should have value <github_scm.name>/<path_from_repository_root>.
+# github_scm.name is specified in the job configuration (next section).
+build_file: "flutter-intellij-kokoro/kokoro/macos_external/kokoro_build.sh"
```

Comment:
`flutter-intellij/kokoro/macos_external/kokoro_build.sh` ?

Reply:
From reading the comment, I don't see `github_scm.name` specified?

> github_scm.name is specified in the job configuration (next section)

Reply:
That quote is from the sample file generated by the code lab. The next section of the code lab describes the job configuration. I should clean that up.